website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -1,29 +1,42 @@
+---
+description: Follow the steps to upgrade the Appsmith installation to a Business Edition.
+---
 # Upgrade to Business Edition
 
-Follow the below guides to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)** based on the type of your **Appsmith Installation:**
+Follow the steps below to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)**. Choose the type of  **Appsmith Installation** to get started.
 
-:::note
-You'll need your  [**Business Edition License Key**](https://www.appsmith.com/pricing)  to complete the upgrade.
+:::info License Key
+You would need a **Business Edition License Key** to complete the upgrade. Learn more on [**Business Edition & pricing**](https://www.appsmith.com/pricing) options.
```

Comment:
would here is a polite request. Linter doesn't allow using will.

What do you think about the reworded sentence below:
`Keep the License Key ready to complete the upgrade. Learn more on [Business Edition & pricing](https://www.appsmith.com/pricing) options.`

Reply:
Can we keep it unless the readability score is showing otherwise

> A License Key is required to complete the upgrade. Learn more about [Business edition & pricing](https://www.appsmith.com/pricing) options.